frontend/src/metabase/query_builder/components/view/ViewHeader.unit.spec.js

```diff
@@ -94,32 +96,37 @@ function mockSettings({ enableNestedQueries = true } = {}) {
 function setup({
   question,
   settings,
-  isRunnable = true,
   isActionListVisible = true,
   isAdditionalInfoVisible = true,
+  isDirty = false,
+  isRunnable = true,
   ...props
 } = {}) {
   mockSettings(settings);
 
   const callbacks = {
     runQuestionQuery: jest.fn(),
+    updateQuestion: jest.fn(),
     setQueryBuilderMode: jest.fn(),
     onOpenModal: jest.fn(),
     onAddFilter: jest.fn(),
     onCloseFilter: jest.fn(),
     onEditSummary: jest.fn(),
+    onOpenQuestionInfo: jest.fn(),
     onCloseSummary: jest.fn(),
     onSave: jest.fn(),
   };
```

Comment:
In this whole file we're just adding some props that are marked as required in `ViewHeader` to remove console errors